pkg/packer/manifests/vsphere/base/linux/centos/http/7/ks.cfg

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Where does this file come from?

Reply:
the kickstart is coming from `konvoy-image` upstream project. The vsphere capv upstream team is using those to provide community OVAs for CAPV.

https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi/packer/ova/linux/centos/http/7